plugins/funind/glob_term_to_relation.ml

```diff
@@ -371,6 +371,7 @@ let add_pat_variables sigma pat typ env : Environ.env =
   in
   let new_env = add_pat_variables  env pat typ in
   let res =
+    let sigma = Evd.from_env new_env in
```

Comment:
This looks wrong, as some printers below print in `env` with this `sigma`.

Reply:
Yup let me have a look, this was some tricky part of the rebase.

Reply:
Yup indeed, that slipped in, thanks.